package-lock.json

```diff
@@ -18,7 +18,7 @@
                 "@deriv-com/quill-tokens": "^2.0.4",
                 "@deriv-com/quill-ui": "^1.10.19",
                 "@deriv-com/translations": "^1.3.2",
-                "@deriv-com/ui": "^1.14.5",
+                "@deriv-com/ui": "^1.28.3",
```

Comment:
Updated package version to add fix for this issue: https://app.clickup.com/t/20696747/WALL-4457

Before:
<img width="1338" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/edaf1bc3-ced0-4586-82c9-1e2ab5921825">

After: 
<img width="1719" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/b467b287-c416-49e0-a483-bad351844fec">